logic/account.js

```diff
@@ -177,25 +118,25 @@ function Account (db, schema, logger, cb) {
 			expression: '("balance")::bigint'
 		},
 		{
-			name: 'u_balance',
+			name: 'votes',
 			type: 'BigInt',
 			filter: {
 				required: true,
 				type: 'integer',
 				minimum: 0,
-				maximum: constants.totalAmount
+				maximum: constants.activeDelegates
 			},
 			conv: Number,
-			expression: '("u_balance")::bigint'
+			expression: '("votes")::int'
```

Comment:
That will work?

Reply:
Yes as discussed it comes from accounts_list view.